docs/middleware.rst

```diff
@@ -28,7 +28,8 @@ Default Middleware
 
 Some middlewares are added by default if you do not supply any. The defaults
 are likely to change regularly, so this list may not include the latest version's defaults.
```

Comment:
The defaults aren't actually changing that much anymore. I think that fact, plus the note to update documentation in `manager.py`, is enough to reduce this to something like:

These middlewares are added by default if you don't add any: